source/jsonnet/england-wales/household/blocks/who-lives-here/anyone_else_list_collector.jsonnet

```diff
@@ -169,13 +163,8 @@ local editQuestion(questionTitle) = {
             type: 'Radio',
             options: [
               {
-                label: {
-                  text: 'Yes, I want to add {ordinality} person',
-                  placeholders: [
-                    placeholders.getListOrdinality('household'),
-                  ],
-                },
-                value: 'Yes, I want to add {ordinality} person',
+                label: 'Yes, I need to add a person',
```

Comment:
`Note - only the variant based on empty list is incorrect and to be changed`

Reply:
I hoping they mean for both, as this is a pain, that would give us 2 values for add person, which I am not sure we can currently do. At least I can't see it anywhere else

Reply:
We might have a problem then.